pdf-bygger/Dockerfile

```diff
@@ -7,14 +23,13 @@ RUN groupadd -r --gid 1001 apprunner && useradd -r --uid 1001 -g apprunner appru
 ENV DEBIAN_FRONTEND="noninteractive" TZ="Europe/London"
 ENV PATH="${PATH}:/app/tex/bin/x86_64-linux/"
 ENV TEXINPUTS=".:/app/pensjonsbrev_latex//:"
+RUN apt update && apt -y install tzdata perl-tk fontconfig && rm -rf /var/lib/apt/lists/*
+#To update TeXLive/XeLaTeX version, run the github workflow and replace the --from argument to newest release
+COPY --from=ghcr.io/navikt/pensjonsbrev/pensjon-latex:2025-05-20-14_26 /app /app
```

Comment:
Nysgjerrig på hvordan du kan endre tag på pensjon-latex her når det kommer endringer på det i denne PR'en?

Reply:
Ja, kva er den beste måten å gjera det på, mon tru? Det eg har gjort undervegs i PR-en er å køyre byggejobben på denne greina, det er jo det enklaste, men prinsipielt kjens det betre å køyre den byggejobben på main etter merge.

Reply:
Tror du må dele det i to PRs?

Reply:
Det er nok det ryddigaste, iallfall. Testar litt i dev først og splittar så opp i to.